lib/util/concerns/hud_lists_2024.rb

```diff
@@ -2371,7 +2363,8 @@ def dependent_under_6_options
       {
         0 => 'No',
         1 => 'Youngest child is under 1 year old',
-        2 => 'Youngest child is 1 to 6 years old and/or one or more children (any age) require significant care',
+        2 => "Youngest child is 1 to 6 years old and/or one or more children (any age) require significant
+care",
```

Comment:
Hmm, is there a `\n` in there somewhere.  Maybe we should strip newlines from the content, I don't think any of the values should contain them.